pymatbridge/publish.py

```diff
@@ -1,5 +1,10 @@
-import IPython.nbformat.v4 as nbformat
-from IPython.nbformat import write as nbwrite
+try:
+    import nbformat.v4 as nbformat
+    from nbformat import write as nbwrite
+except:
```

Comment:
Please explicitly use `except ImportError`, to avoid hiding any other kind of error.